docker-compose.prod.yml

```diff
@@ -79,6 +79,7 @@ services:
       target: runtime
       secrets:
         - gh-token
+    user: root
     volumes:
       - calimero_auth_data:/data
```

Comment:
it's worth mentioning this is due to the volume being owned by the host, but used by the guest, so we have a priviledge mismatch

this is okay for now, but a better solution should eventually be devised